app/pages/docs/tutorial.mdx

```diff
@@ -978,24 +981,26 @@ export default resolver.pipe(
 ```
 
 [`upsert`](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#upsert)
-is a special operation that means, "If this item exists, update it. Else
-create it". This is perfect for this case because we didn't require the
-user to add three choices when creating the question. So if later the user
-adds another choice by editing the question, then it'll be created here.
+is een speciale operatie die betekent: "Als dit item bestaat, update het
```

Comment:
```suggestion
is een speciale handeling die betekent: "Als dit item bestaat, update het
```

Leek mij duidelijker in deze context. Operatie doet me meer aan de ziekenhuisbehandeling denken.

Reply:
Misschien hier **database operatie** van maken? De term operatie wordt in deze context wel gebruikt in onze sector natuurlijk. Handeling vind ik wat vreemder.

Reply:
Ja databaseoperatie is nog beter.  Ben niet zeker van de spatie. Beide variaties worden blijkbaar gebruikt.

Reply:
Google wilt "databaseoperatie" altijd veranderen naar "database operatie" 😄 Ik vind het af elkaar nog iets duidelijker om te lezen. Een samenstelling met twee klinkers tegen elkaar vind ik er vaak wat moeilijk uitzien.